netty/src/main/java/io/grpc/transport/netty/ProtocolNegotiators.java

```diff
@@ -124,16 +117,7 @@ public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
             SSLParameters sslParams = new SSLParameters();
             sslParams.setEndpointIdentificationAlgorithm("HTTPS");
             sslEngine.setSSLParameters(sslParams);
-
-            SslHandler sslHandler = new SslHandler(sslEngine, false);
-            sslHandler.handshakeFuture().addListener(new GenericFutureListener<Future<Channel>>() {
-              @Override
-              public void operationComplete(Future<Channel> future) throws Exception {
-                // If an error occurred during the handshake, throw it to the pipeline.
-                future.get();
-              }
-            });
-            ctx.pipeline().replace(this, null, sslHandler);
+            ctx.pipeline().replace(this, null, new SslHandler(sslEngine, false));
```

Comment:
I don't recall exactly why the previous code was there ... is it no longer needed?